plugins/engines/optix/CMakeLists.txt

```diff
@@ -28,8 +28,8 @@ set(BRAYNSOPTIXENGINEPLUGIN_PUBLIC_HEADERS
 )
 
 set(BRAYNSOPTIXENGINEPLUGIN_LINK_LIBRARIES
-    PUBLIC vmmlib braynsCommon braynsIO
-    PRIVATE braynsCommonEnginePlugin ${OptiX_LIBRARIES}
+  PUBLIC vmmlib braynsCommon braynsIO
+  PRIVATE braynsCommonEnginePlugin Lunchbox ${OptiX_LIBRARIES}
```

Comment:
correct - will do if final solution uses Lunchbox, but waiting for @favreau who is looking for an alternative